packages/wrangler/src/pages/dev.tsx

```diff
@@ -1,17 +1,19 @@
 import { execSync, spawn } from "node:child_process";
-import { existsSync } from "node:fs";
+import { existsSync, readFileSync } from "node:fs";
 import { homedir, tmpdir } from "node:os";
 import { join, resolve } from "node:path";
 import { watch } from "chokidar";
-import { build as workerJsBuild } from "esbuild";
+import { build, build as workerJsBuild } from "esbuild";
```

Comment:
Bit of a weird import here. Did you mean to import the same thing twice like this?

Reply:
ah, so glad you caught this. I saw it as well and was confused as to why it is this way, but didn't fix immediately so here we are are. I think some auto import thing went terribly wrong here. will fix

Reply:
✅